paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/PaymentSelection.kt

```diff
@@ -16,9 +16,12 @@ import com.stripe.android.paymentsheet.paymentdatacollection.ach.ACHText
 import com.stripe.android.paymentsheet.paymentdatacollection.ach.USBankAccountFormScreenState
 import kotlinx.parcelize.IgnoredOnParcel
 import kotlinx.parcelize.Parcelize
+import com.stripe.android.ui.core.R as StripeUiCoreR
 
 internal sealed class PaymentSelection : Parcelable {
 
+    var hasAcknowledgedSepaMandate: Boolean = false
```

Comment:
Is this the right place to have sepa mandate check? Would it make sense to make this more generic? Maybe even nullable?

Reply:
Sepa is kinda like US Bank, very one off. I talked with Yuki about this too, while a generic solution seems reasonable, sepa is pretty unique in all of this.

This felt like the best place to put this when I was developing it, but if there's a better place, I'm more than happy to move it.